src/mem/external_master.hh

```diff
@@ -70,12 +70,17 @@ class ExternalMaster : public SimObject
         ExternalMaster &owner;
 
       public:
+        /**
+         * @ingroup api_port
```

Comment:
External ports are different from the "port" API. Maybe this should be "external_api"?

Reply:
I agree.

Reply:
Done